docs/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@nuxt/docs",
+  "version": "4.0.0-0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nuxt/nuxt.git",
+    "directory": "packages/docs"
```

```suggestion
    "directory": "docs"
```

Shouldnt this be `docs`?